### PR TITLE
Correct some spellings and grammars

### PR DESCRIPTION
**Pull request by yfpeng**

### CODE REVIEW

File: sections/02_intro.md [hunk 1]
```diff
@@ -116,11 +116,11 @@ application of deep learning.
 A key challenge in biomedicine is the accurate classification of
 diseases and disease subtypes. In oncology, current "gold standard"
 approaches involve histology, requiring manual
-human expertise for quantification, or small panel of molecular markers,
+human expertise for quantification, or small panels of molecular markers,
 such as cell surface
 receptors or genes' expression. One example is the PAM50 approach
 to classifying breast cancer where the expression of 50 marker
-genes divides breast cancer patients into four subtypes.
+genes divide breast cancer patients into four subtypes.
```
Comment (agitter) [inline sections/02_intro.md]: I think "divides" is correct.  The noun is the singular "expression".

File: sections/03_categorize.md [hunk 2]
```diff
@@ -540,7 +540,7 @@ adversarial neural networks as a tool to make sharable EHR data
 model from such attacks.
 
 There are approaches to protect models, but they pose their own challenges.
-Training algorithms in a differentially private manner provides a limited
+Training algorithms in a differentially private manner provide a limited
```
Comment (agitter) [inline sections/03_categorize.md]: Keep "provides".  "Training" is singular.

File: sections/04_study.md [hunk 3]
```diff
@@ -351,13 +351,13 @@ necessarily operate on the nearest gene and may in fact effect multiple genes.
 Their activity is frequently tissue- or context-specific. A substantial fraction
 of enhancers displays modest or no conservation across species. There is no
 universal enhancer sequence signal or marker for enhancers, and some literature
-suggests that enhancers and promoters may be just categories along a spectrum
+suggests that enhancers and promoters may be just categories along with a spectrum
```
Comment (agitter) [inline sections/04_study.md]: It read better before

File: sections/05_treat.md [hunk 4]
```diff
@@ -172,7 +172,7 @@ creating a DBN of stacked RBMs called DeepDTIs, which is able to predict
 interactions on the basis of chemical structure and protein sequence features.
 
 Drug repositioning appears to be an obvious candidate for the development of
-deep learning applications both because of the large amount of high-dimensional
+deep learning applications both because of a large amount of high-dimensional
```
Comment (agitter) [inline sections/05_treat.md]: I prefer "the"

File: sections/05_treat.md [hunk 5]
```diff
@@ -364,7 +364,7 @@ advantages of jointly modeling many targets [@tag:Unterthiner2014_screening
 @tag:Ramsundar2015_multitask_drug].  Other studies have shown the limitations of
 transfer learning when the prediction tasks are insufficiently related
 [@tag:Kearnes2016_admet @tag:AltaeTran2016_one_shot].  This has important
-implications for representation learning.  The typical approach of improving
+implications for representation learning.  The typical approach to improving
```
Comment (agitter) [inline sections/05_treat.md]: I prefer "of improving" or "to improve"

File: sections/03_categorize.md (hunk 2 above)
Comment (yfpeng) [inline sections/03_categorize.md]: The subject is "algorithms"

Comment (agitter) [inline sections/03_categorize.md]: I still think "training".  Remove "algorithms" from the sentence and it still works:

> Training in a differentially private manner provides a limited...

**Comment by yfpeng** [comment]
All fixed.

**Comment by agitter** [comment]
Thanks, merged.

**Pull request merged**